### ISSUE TITLE
Maintain lists over projects that do/don't want PRs made with Sorald

### ISSUE THREAD

**Issue by slarse** [SpoonLabs/sorald#388]
As we're starting to make PRs, we find that some projects are more open to PRs made with Sorald than others. I think we should maintain a list of projects that are open to PRs, and projects that clearly aren't. The latter I think is important so as not to offend project maintainers.

@monperrus @nharrand @khaes-kth @fermadeiral thoughts?

**Comment by monperrus** [comment]
Maintaining a list of projects who care about Sonar is an excellent idea!

**Comment by khaes-kth** [comment]
@slarse Good idea! Should we add a new documentation file?

**Comment by slarse** [comment]
@khaes-kth Yeah, that sounds good, we could just make it a simple Markdown file with two headers, one for "like sonar" and one for "don't like sonar". It initially only needs to be human-readable, if we need machine-readable we can adjust it later.

Could you draft a first version as you've got most of that data?
